packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -822,15 +834,39 @@ async function formNestedStack(
   serviceName?: string,
   skipEnv?: boolean,
 ) {
-  const initTemplateFilePath = path.join(__dirname, '..', 'resources', rootStackFileName);
-  const nestedStack = JSONUtilities.readJson<Template>(initTemplateFilePath);
+  let rootStack;
+  if (FeatureFlags.getBoolean('overrides.project')) {
```

Comment:
Don't think we even need this check for FF for root-stack since right now its a transient runtime file which is not checked in. So with this change, we would be placing this root-stack in a build sub-folder and uploading it to current-cloud..which is fine since it's not a breaking change.

Reply:
ill check this out in migration test for this